.github/workflows/code.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: "ubuntu-latest"
     steps:
       - name: Run black, flake8, mypy
-        uses: dls-controls/pipenv-run-action@v1
```

Comment:
We'll have to do this action repo first when we come to migrate everything

Reply:
Or last? The PRs to change `dls-controls` to `DiamondLightSource` won't come into effect until someone merges them

Reply:
yeah, but they'll all fail CI if we don't move the action first...

Reply:
I guess it's whether we want them pass the `main` CI or the `migrate-dls-controls-diamondlightsource` CI. Probably the latter

Reply:
Hopefully the github git repo redirection will mean that both can pass...

Reply:
Hadn't considered that for actions, hopefully :crossed_fingers: